frontend/src/components/Collection/components/CollectionDescription/index.tsx

```diff
@@ -42,6 +39,13 @@ export default function CollectionDescription({
     });
   };
 
+  /**
```

Comment:
Currently, the description `ref` assignment occurs on component rerenders.

Because our network requests are now reduced to a minimum, we need to create an opportunity within the component for the `ref` to be assigned to the description. By changing the initial `ellipsisMode` to `undefined` and then, on component mount, setting the `ellipsisMode` to "NONE" we are effectively creating the opportunity for the `ref` to be assigned, ready for use by the resize observer hook!

Reply:
Oooh really great insight! Can we add a summary of this to code comment please?

Something like the following or better: 

```ts
* Ellipsis mode state is initialized to "NONE" only when the component mounts,
* so we guarantee that `descriptionRef` gets the element handle for resize observer hook.
```